### PR TITLE
docs(chart): remove images

### PR DESCRIPTION
**Pull request by giatsidis**
- Removed images and captions of chart component examples

### CODE REVIEW

**Comment by marin-bratanov** [comment]
Is there actual feedback that people prefer to click and wait for a repl to run rather than see what is expected immediately?

**Comment by svdimitr** [comment]
@marin-bratanov, In the kendo-themes release they changed the primary colors of the charts, thus all screenshots are wrong. That and the situation with the component options (we will release them with one visual, but they will change right in the next release) led us to conclude that having so many screenshots in the docs is not feasible. So we will gradually remove all screenshots, apart from the ones that explain the structure of the component (as in StockChart, ColorPicker, etc.).

**Review by svdimitr — APPROVED** [review]

File: components/chart/events.md [hunk 1]
```diff
@@ -177,9 +173,7 @@ These examples showcase the different applications of the `OnSeriesClick` event.
 }
 ````
 
->caption The result from the code snippet above
-
-![onseriesclick get data model example](images/onseries-click-get-model-example.gif)
+![onseriesclick get data model example]
```
Comment (ntacheva) [inline components/chart/events.md:176]:
```suggestion
```

File: components/chart/types/area.md [hunk 2]
```diff
@@ -15,12 +15,6 @@ The <a href="https://www.telerik.com/blazor-ui/area-chart" target="_blank">Blazo
 
 An Area chart emphasizes the volume of money, data or any other unit that the given series has encompassed. When backgrounds are semi-transparent, it lets the user clearly see where different sets of data overlap.
```
Comment (ntacheva) [inline components/chart/types/area.md:17]:
```suggestion

@[template](/_contentTemplates/chart/link-to-basics.md#understand-basics-and-databinding-first)

```

**Review by ntacheva — APPROVED** [review]

**Review by xristianstefanov — APPROVED** [review]

**Pull request merged**